doorstop/core/files/doorstop.css

```diff
@@ -1,6 +1,6 @@
 body{
     margin: 0 auto;
-    font-family: Georgia, Palatino, serif;
+    font-family: "Helvetica Neue",Helvetica,Arial,sans-serif;
```

Comment:
See my comments https://github.com/doorstop-dev/doorstop/pull/371#issuecomment-513960855 about this file. I *think* we can get rid of `doorstop.css` or merge it with `general.css`?

Reply:
I agree. We got rid of the call to _lines_css() which was the only place this css is currently used as far as I can tell. I'll update the PR the remove the file and any unused references to it.